ws/Cargo.toml

```diff
@@ -12,4 +12,4 @@ documentation = "https://paritytech.github.io/jsonrpc/json_ws_server/index.html"
 log = "0.3"
 jsonrpc-core = { version = "7.0", path = "../core" }
 jsonrpc-server-utils = { version = "7.0", path = "../server-utils" }
-ws = { git = "https://github.com/tomusdrw/ws-rs" }
+ws = { git = "https://github.com/faern/ws-rs", branch = "arbitrary-handshake-responses" }
```

Comment:
Yup. This is one of the reasons I suggested you do that :) So you don't have to trust my repo.